GoogleForCreators/web-stories-wp
# Design panel behavior and state

## Summary
Disables persisting for Style panels so that these would remain open when switching elements/adding a new even if the user has collapsed it.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
I
1. I add text to the page. Verify the **Style** section is defaulted open, all the other sections are collapsed.
2. I add a shape to the page. Verify the **Style** section is defaulted open. 



II
1. I add text to the page. I expand the **Layer** and Link **panel**
2. I add another text element to the page.
3. Verify that Style section is defaulted open while **Layer** and **Link** panels should remain open
4. I close the Link panel.
5. I add a shape element to the page. 
6. Verify that Style section is defaulted open while **Layer** panel should remain open



III
1. I add text to the page. The Style section is defaulted open. 
2. I collapsed the Style section
3. I add another text box to the page.
4. Verify the **Style** section is defaulted open.


---



Fixes #4996
